sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+    "except Exception as ex:\n",
+    "    print(ex)\n",
+    "    # Enter details of your AML workspace\n",
+    "    subscription_id = \"<SUBSCRIPTION_ID>\"\n",
+    "    resource_group = \"<RESOURCE_GROUP>\"\n",
+    "    workspace = \"<AML_WORKSPACE_NAME>\"\n",
+    "    ml_client = MLClient(credential, subscription_id, resource_group, workspace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "source": [
+    "### Show Azure ML Workspace information"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "outputs": [],
+   "source": [
+    "workspace = ml_client.workspaces.get(name=ml_client.workspace_name)\n",
+    "\n",
+    "output = {}\n",
+    "output[\"Workspace\"] = ml_client.workspace_name\n",
+    "output[\"Subscription ID\"] = ml_client.connections._subscription_id\n",
+    "output[\"Resource Group\"] = workspace.resource_group\n",
+    "output[\"Location\"] = workspace.location\n",
+    "output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "source": [
+    "# 2. Data\n",
+    "\n",
+    "We will use github active user (DAU) count for model training. The data is stored in a tabular format.\n",
+    "\n",
+    "With Azure Machine Learning MLTables you can keep a single copy of data in your storage, easily access data during model training, share data and collaborate with other users. \n",
+    "Below, we will upload the data by creating an MLTable to be used for training.\n",
+    "\n",
+    "**NOTE:** In this PRIVATE PREVIEW we're defining the MLTable in a separate folder and .YAML file.\n",
+    "In later versions, you'll be able to do it all in Python APIs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "hideCode": false,
+    "hidePrompt": false
+   },
+   "outputs": [],
+   "source": [
+    "from helpers.generate_ml_table import create_ml_table\n",
+    "\n",
+    "create_ml_table(\"github_dau_2011-2018_train.csv\", \"./data/training-mltable-folder\")\n",
+    "\n",
+    "# Training MLTable defined locally, with local data to be uploaded\n",
+    "my_training_data_input = Input(\n",
+    "    type=AssetTypes.MLTABLE, path=\"./data/training-mltable-folder\"\n",
+    ")\n",
+    "\n",
+    "import os\n",
+    "import shutil\n",
+    "\n",
+    "os.makedirs(\"test_dataset\", exist_ok=True)\n",
+    "shutil.copy(\n",
+    "    \"github_dau_2011-2018_test.csv\",\n",
+    "    \"test_dataset/github_dau_2011-2018_test.csv\",\n",
+    ")\n",
+    "\n",
+    "my_test_data_input = Input(\n",
+    "    type=AssetTypes.URI_FOLDER,\n",
+    "    path=\"test_dataset/\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3. Configure and run the AutoML Forecasting training job\n",
```

Comment:
Let's stick with uniform sections across forecasting notebook. Please change section 3 to be the same as in the rest of the notebooks and titled `3 Create or Attach existing AmlCompute`. Sections 4 should be `4. Configure and run the AutoML Forecasting training job`. Doing this will reduce the maintenance cost of these notebooks as well.

Reply:
Section added.